src/loginServices.js

```diff
@@ -2,8 +2,9 @@ import _ from 'lodash';
 import 'isomorphic-fetch';
 import localforage from 'localforage';
 import { change } from 'redux-form';
-import { addLocaleData } from 'react-intl';
+import { addLocaleData, FormattedMessage } from 'react-intl';
 import { translations } from 'stripes-config';
+import React from 'react';
```

Comment:
React import usually placed on the very top. Please move it there for consistency.